docs/knowledge_base/policy_development.md

```diff
@@ -108,3 +138,130 @@ If policy evaluation is successful, the result will contain both the policy outp
 Issues and dependencies that have been suppressed via project preferences are visible in the policy input, but rejections related to those issues or dependencies will not be included in the Markdown report.
 
 Dependencies that are ignored via the `ignored_packages` parameter are filtered out before applying the policy and will not be visible in the policy input or output.
+
+# Policy Examples
+
+The policy transforms your threat model into a description of why the job is being blocked. There are multiple ways to define why a job should be blocked.
+
+The `METADATA` block contains `OPA` [Annotations](https://www.openpolicyagent.org/docs/latest/annotations/) which correlate to the schema and can be used for type checking.
+
+## Blocking an issue
+
+The most common reason to block a job is because one of the dependencies has a known issue within one of Phylum's risk domains.
+
+The following policy shows ways to block using an `issue` rule based on a per-domain threshold.
+
+```rego
+package policy.v1
+
+import data.phylum.domain
+import data.phylum.level
+import rego.v1
+
+# METADATA
+# title: risk level cannot exceed medium
+deny contains issue if {
+    some issue in data.issues
+    issue.domain in {domain.AUTHOR, domain.ENGINEERING, domain.VULNERABILITY}
+    issue.severity > level.MEDIUM
+}
+
+# METADATA
+# title: malicious risk level cannot exceed low
+deny contains issue if {
+    some issue in data.issues
+    issue.domain == domain.MALICIOUS
+    issue.severity > level.LOW
+}
+
+# METADATA
+# title: license risk level cannot exceed high
+deny contains issue if {
+    some issue in data.issues
+    issue.domain == domain.LICENSE
+    issue.severity > level.HIGH
+}
+```
+
+Given the following input:
+
+```json
+{
+    "issues": [{
+        "id": "b8ad4443-d875-427b-9eda-b4b2fb1d6212",
+        "domain": "malicious",
+        "severity": 4,
+        "tag": "CM0004"
+    }]
+}
+```
+
+When the policy fails, the output will look something like this:
+
+```json
+{
+    "deny": [{
+        "id": "b8ad4443-d875-427b-9eda-b4b2fb1d6212",
+        "domain": "malicious",
+        "severity": 4,
+        "tag": "CM0004"
+    }]
+}
+```
+
+When Phylum sees this output from the policy, it will block the job and generate a report naming the package and describing the issue.
+
+## Blocking a dependency
+
+You may also block on a dependency-level characteristic using a `dependency` rule.
+
+The following policy blocks packages belonging to a namespace.
+
+```rego
+package policy.v1
+
+import rego.v1
+
+# METADATA
+# title: AGPL licensed software is not allowed.
+deny contains dependency if {
+    some dependency in data.dependencies
+    regex.match("(?i)\\bAGPL\\b", dependency.license)
+}
```

Comment:
I think this is still a good example, but we might want to call out that we have a similar policy that detects copyleft licenses in general.

Reply:
added a note

Reply:
also - rejection based on this type of policy will not be suppressible i'm afraid, because it lacks a tag. the only way to suppress this would be via ignoring the package itself - which is something we agreed not to do anymore

Reply:
Understood. That should be remedied when this [issue](https://github.com/phylum-dev/api/issues/1446) is complete.